test_util/cluster.py

```diff
@@ -184,6 +202,86 @@ def check_ssh(self, hosts):
         """
         self.ssher.remote_cmd(hosts, ['echo'])
 
+    def mesos_metrics_snapshot(self, host):
+        """Return a snapshot of the Mesos metrics for host."""
+        if host in self.masters:
+            port = 5050
+        else:
+            port = 5051
+
+        with self.ssher.tunnel(host) as tunnel:
+            return json.loads(
+                tunnel.remote_cmd(
```

Comment:
Not all clusters have a bootstrap host, so if it's all the same then this is method is more general.